src/main/java/graphql/introspection/Introspection.java

```diff
@@ -430,13 +430,17 @@ private static String print(Object value, GraphQLInputType type) {
             .name("__Directive")
             .field(newFieldDefinition()
                     .name("name")
-                    .type(GraphQLString))
+                    .description("The __Directive type represents a Directive that a server supports.")
+                    .type(nonNull(GraphQLString)))
             .field(newFieldDefinition()
                     .name("description")
                     .type(GraphQLString))
+            .field(newFieldDefinition()
+                    .name("isRepeatable")
+                    .type(nonNull(GraphQLBoolean)))
             .field(newFieldDefinition()
                     .name("locations")
-                    .type(list(nonNull(__DirectiveLocation))))
+                    .type(nonNull(list(__DirectiveLocation))))
```

Comment:
what is this fix here? did we have this wrong before?

Reply:
let me check - this came from other code that I started this branch from - a PR from @dugenkui03  I think.

Reply:
The spec is not explicit

> `locations` returns a List of __DirectiveLocation representing the valid locations this directive may be placed.

So I will make it

```
.field(newFieldDefinition()
                    .name("locations")
                    .type(nonNull(list(nonNull(__DirectiveLocation)))))
```

Reply:
The spec is explicit (see https://spec.graphql.org/draft/#sec-Schema-Introspection):
```
type __Directive {
  name: String!
  description: String
  locations: [__DirectiveLocation!]!
  args: [__InputValue!]!
  isRepeatable: Boolean!
}
```